README.md

```diff
@@ -144,7 +144,7 @@ steps:
 - Setting "Keep this build forever" for the check-merge Jenkins build.
 
 - Updating releng-tools with the new VDSM version.  See releng-tools
-  repo (`git clone https://gerrit.ovirt.org/releng-tools`) and VDSM
+  repo (`git clone https://github.com/oVirt/releng-tools`) and VDSM
   related patches there for examples.
```

Comment:
The change is good, but it will be more useful if we link to a previous patch showing showing how the change looks like. Here is how we do this in imageio:
https://github.com/oVirt/ovirt-imageio/blob/master/RELEASE.md#upstream